README.md

```diff
@@ -71,6 +71,16 @@ The recommended method to install _Elasticsearch-PHP_ is through [Composer](http
         require 'vendor/autoload.php';
 
         $client = ClientBuilder::create()->build();
+        
+        // OPTIONALLY set the hosts on the 'create'
+        // defaults to localhost
+        // $client = ClientBuilder::create()
+        //      ->setHosts(
+        //        [
+        //          'your-elastic-host.com'
+        //        ]
+        //      )
+        //      ->build();
```

Comment:
this is really useful but would this be better as a separate example rather than a comment?